### ISSUE TITLE
Video controls are not read and focused in full screen mode when android talkback is on. 

### ISSUE THREAD

**Issue by AnkurOO9** [sampotts/plyr#2294]
Ionic version:
[x] 5.x

Current behaviour:
Video controls are not read and focused in full screen mode when android talkback is on. 

Expected behaviour:
Video controls should read and should be focused in full screen mode when talkback is on.

Steps to reproduce:
1) Turn on talkback.
2) Play any video. Double tap on enter full screen. and tap on any video control.
